clash-prelude/src/Clash/Signal.hs

```diff
@@ -373,37 +384,38 @@ f :: 'HiddenClockResetEnable' dom
   -> Signal dom Int
 @
 
-which has hidden clock and routed reset arguments, we expose those hidden
-arguments so that we can explicitly apply them:
```

Comment:
Does `routed` have some special meaning here?
My feeling is to just say "... hidden clock, reset and enable arguments"

Reply:
I have no idea, I thought maybe it had a special meaning, so I kept it.

Reply:
It got shortened from "implicitly routed clock/reset" in [this big patch](https://github.com/clash-lang/clash-compiler/commit/26973e75d65e9de4942d298e4b55f2cec3e7b7f3#diff-3997ddd6d1b20d5deb35e5fd5056f8fc31ca6a8826afa19ac9c1982b5795a832L217-L218):
```diff
-which has an implicitly routed clock and implicitly routed clock, we can use
-it in a function with explicit clock and reset arguments like so:
+which has hidden clock and routed reset arguments, we expose those hidden
+arguments so that we can explicitly apply them:
```
So you can remove the "routed".
It's meaningless without the "implicitly".

Reply:
Oh thanks for the archeology! It hadn't occurred to me to do that!